resource-server/server.js

```diff
@@ -0,0 +1,83 @@
+const express = require('express');
+const OktaJwtVerifier = require('@okta/jwt-verifier');
+var cors = require('cors');
+
+const config = require('./.samples.config.json').oktaSample;
+
+const oktaJwtVerifier = new OktaJwtVerifier({
+  issuer: config.oidc.issuer,
+  assertClaims: {
+    aud: 'api://default',
```

Comment:
We also need to check for the `cid` claim. This should be pulled from `.samples.config.json`.